docs/ref/migration-operations.txt

```diff
@@ -301,6 +321,7 @@ model::
         operations = [
             migrations.RunPython(
```

Comment:
I'd consolidate this on one line: `migrations.RunPython(forwards_func, reverse_func)`